core/src/main/clojure/xtdb/sql/plan.clj

```diff
@@ -393,6 +393,14 @@
        ;; should never be reached
        (throw (err/illegal-arg :xtdb.sql/parse-error {::err/message "Invalid Period Predicand"}))))))
 
+(defn routine-invocation [_schema routine-name _fn-opts]
```

Comment:
Essentially ignoring both schema and shape of SQL args here - _could- make it check schema if present but not sure that really adds anything for us?